nfd-worker-daemonset.yaml.template

```diff
@@ -94,6 +98,10 @@ spec:
         - name: nfd-worker-conf
           configMap:
             name: nfd-worker-conf
+## Example for more custom configs in an additional configmap (2/3)
+#        - name: more-custom-configs
+#          configMap:
+#            name: more-custom-configs
```

Comment:
For consistency, add this to the "combined" and "job" templates, too

Reply:
done